bin/diff-before-update

```diff
@@ -51,3 +51,19 @@ compare files/$folder/config/user_preferences_extra_conf.yml	$GALAXY/config/user
 #compare templates/$folder/config/build_sites.yml.j2	$GALAXY/config/build_sites.yml.sample.sample
 
 compare group_vars/gxconfig${ext}.yml	$GALAXY/config/galaxy.yml.sample
+
+# Compare file source templates
+compare files/$folder/config/file_source_templates/production_aws_private_bucket.yml	$GALAXY/files/templates/examples/production_aws_private_bucket.yml
```

Comment:
Any idea how we remember to add new templates here if we add new ones?

Reply:
I will update our operations [here](https://github.com/usegalaxy-eu/operations/blob/main/upgrade.md) and add additional checkpoints there that this script should updated if and when new templates from upstream are added.

Reply:
Additionally, we can loop over all the files in our directory and compare them to the files in the respective templates dir in the upstream and compare them during the upgrade cycle.

Reply:
Could this be done with a github action that checks what upsteam has and compares it to master and then fails if not in sync?

Reply:
unfortunately not because the upstream has more templates; we picked only the ones that are "production" ready.

Reply:
Documentation [PR](https://github.com/usegalaxy-eu/operations/pull/46)